src/assets/js/search.js

```diff
@@ -135,6 +137,11 @@
   const setHeading = (resultsCount, searchQuery) => {
     const searchHeading = document.getElementById('search-heading')
 
+    if (!searchQuery) {
```

Comment:
The second `<h1>` is added in `search.html`, unless you mean that some other part of this method is adding something? Then that first Jekyll-autogenerated `<h1>` is removed at the end of the method by `removeAutogeneratedHeading`. axe did complain about having multiple `<h1>`s, which is the reason for adding `removeAutogeneratedHeading`, beyond DOM hygiene

Reply:
Ah! If you land on `Enter a search term` because there's no query, you get the duplicate because of returning before `removeAutogeneratedHeading` is called. Got it!